R/DiseasyBaseModule.R

```diff
@@ -1,6 +1,24 @@
-#' @title Base module for the framework
+#' @title Diseasy' basic module
 #'
-#' @description TODO
+#' @description
+#'   The `DiseasyBaseModule` module implements common functionality that all modules have available.
+#'   Most notably, the base module facilitates:
+#'   * logging:
+#'     A `lgr` logger is configured when making any module that inherits from the base module.
+#'     This logger is stored in `private$lg`.
+#'   * hashing:
+#'     The active binding `self$hash` hashes the values of public fields to provide a way to uniquely identify the
+#'     configuration of modules.
+#'     Care must be taken to ensure that information that makes modules distinct are also stored in public fields.
+#'     That is, if a important property is stored only as private field, the hash will not change.
```

Comment:
Typo: 'That is, if _an_ important property...'

Reply:
and: '... only as _a_ private field, ...'

Reply:
Thanks!